contrib/slow_sql_query/slow_query.txt

```diff
@@ -0,0 +1,40 @@
+1. 介绍
```

Comment:
It is good to see you write introductions. How about translate it into English and write them as a Markdown file?